packages/client-ts/src/context/engines/verida/messaging/outbox.ts

```diff
@@ -68,10 +68,13 @@ class VeridaOutbox {
     did = did.toLowerCase();
 
     const defaults: MessageSendConfig = {
+      did: did,
       // By default send data to the user's official Verida Vault application
       recipientContextName: VAULT_CONTEXT_NAME,
       // @todo: set a default expiry that is configurable but defaults to 24 hours?
     };
+    // TODO: This replaces the `did` specified in config
```

Comment:
Refactor now or in a separate issue later?

Reply:
Addresses in :- https://github.com/verida/verida-js/issues/131